Formula/termtunnel.rb

```diff
@@ -0,0 +1,17 @@
+class Termtunnel < Formula
+  desc "Cross-platform terminal tunnel tool"
+  homepage "https://github.com/beordle/termtunnel"
+  url "https://github.com/beordle/termtunnel/archive/refs/tags/version-1.7.1.tar.gz"
+  sha256 "112b0df87cf23db792d7b654a0554230900a8f269a0f9756a304d129552e8580"
+  license "MIT"
+  depends_on "cmake" => :build
+  def install
+    system "cmake", "."
+    system "make"
+    bin.install "termtunnel"
+  end
+
+  test do
+    system "#{bin}/termtunnel"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.